src/test/kotlin/no/skatteetaten/aurora/gobo/service/WebsealAffiliationServiceTest.kt

```diff
@@ -13,21 +14,29 @@ import org.junit.jupiter.api.Test
 class WebsealAffiliationServiceTest {
 
     private val applicationService = mockk<ApplicationServiceBlocking> {
-        every { getApplications(any(), any()) } returns listOf(ApplicationResourceBuilder().build())
+        every { getApplications(any(), any()) } returns listOf(
+            ApplicationResourceBuilder(affiliation = "paas", namespace = "paas").build(),
+            ApplicationResourceBuilder(affiliation = "aurora", namespace = "aurora").build()
+        )
     }
     private val websealService = mockk<WebsealServiceBlocking> {
         every { getStates() } returns listOf(
-            WebsealStateBuilder(namespace = "namespace").build(),
-            WebsealStateBuilder(namespace = "aurora").build()
+            WebsealStateBuilder(namespace = "paas").build(),
+            WebsealStateBuilder(namespace = "aurora").build(),
+            WebsealStateBuilder(namespace = "test1").build(),
+            WebsealStateBuilder(namespace = "test2").build()
         )
     }
 
     private val websealAffiliationService = WebsealAffiliationService(applicationService, websealService)
 
     @Test
     fun `Get WebSEAL state for affiliation`() {
```

Comment:
Burde det nå være "Get WebSEAL states for affiliations"?

Reply:
enig, I fix